holoviews/core/util.py

```diff
@@ -1776,3 +1776,35 @@ def numpy_scalar_to_python(scalar):
     elif np.issubclass_(scalar_type, np.int_):
         return int(scalar)
     return scalar
+
+
+def closest_match(match, specs, depth=0):
```

Comment:
I'm going to assume ``closest_match`` has been moved but not changed.

Reply:
There doesn't seem to be any unit tests for ``closest_match`` right now. Would be good to have and it looks like the sort of utility that could be easily tested.